src/map/skill.cpp

```diff
@@ -7483,6 +7483,53 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 
 		break;
 
+	case NPC_IMMUNE_PROPERTY:
+		switch (skill_lv) {
+			case 1  : type = SC_IMMUNE_PROPERTY_NOTHING;	break;
+			case 2  : type = SC_IMMUNE_PROPERTY_WATER;		break;
+			case 3  : type = SC_IMMUNE_PROPERTY_GROUND;		break;
+			case 4  : type = SC_IMMUNE_PROPERTY_FIRE;		break;
+			case 5  : type = SC_IMMUNE_PROPERTY_WIND;		break;
+			case 6  : type = SC_IMMUNE_PROPERTY_DARKNESS;	break;
+			case 7  : type = SC_IMMUNE_PROPERTY_SAINT;		break;
+			case 8  : type = SC_IMMUNE_PROPERTY_POISON;		break;
+			case 9  : type = SC_IMMUNE_PROPERTY_TELEKINESIS;break;
+			case 10 : type = SC_IMMUNE_PROPERTY_UNDEAD;		break;
```

Comment:
```suggestion
			case 1:
				type = SC_IMMUNE_PROPERTY_NOTHING;
				break;
			case 2:
				type = SC_IMMUNE_PROPERTY_WATER;
				break;
			case 3:
				type = SC_IMMUNE_PROPERTY_GROUND;
				break;
			case 4:
				type = SC_IMMUNE_PROPERTY_FIRE;
				break;
			case 5:
				type = SC_IMMUNE_PROPERTY_WIND;
				break;
			case 6:
				type = SC_IMMUNE_PROPERTY_DARKNESS;
				break;
			case 7:
				type = SC_IMMUNE_PROPERTY_SAINT;
				break;
			case 8:
				type = SC_IMMUNE_PROPERTY_POISON;
				break;
			case 9:
				type = SC_IMMUNE_PROPERTY_TELEKINESIS;
				break;
			case 10:
				type = SC_IMMUNE_PROPERTY_UNDEAD;
				break;
```
White space is free ;)